commons/src/test/java/com/simpletools/calculator/commons/CalculatorImplTest.kt

```diff
@@ -126,6 +126,72 @@ class CalculatorImplTest {
         assertEquals("4", result)
     }
 
+    @Test
+    fun chainedEqualsUnaryOperationTest() {
+        calc.addDigit(256)
+        calc.handleOperation("root")
+        calc.handleEquals() // No effect expected
+        calc.handleEquals() // No effect expected
+        calc.handleEquals() // No effect expected
+        assertEquals("16", activity.getResult())
+        checkFormula("√256")
+    }
+
+    @Test
+    fun chainedEqualsBinaryOperationTest() {
+        calc.addDigit(10)
+        calc.handleOperation("plus")
+        calc.addDigit(1)
+        calc.handleEquals()     //11
```

Comment:
Are these comments really necessary at this point ?

Reply:
I mean it's a lot of comments but it doesn't hurt

Reply:
I find it helps to understand what the test is expected to do at a glance and without running the test.